### ISSUE TITLE
Rating mouse pointer should be the "clickable/hand" one

### ISSUE THREAD

**Issue by bhathiya** [wso2/product-apim#6619]
![image](https://user-images.githubusercontent.com/3960742/67364571-6ed29880-f58d-11e9-81fc-80412a868e32.png)

**Comment by kavishkafernando** [comment]
Fixed in latest 3.1.0 pack.
